src/SFA.DAS.AdminService.Web/Infrastructure/RoatpClients/RoatpApplicationApiClient.cs

```diff
@@ -21,46 +21,12 @@ public RoatpApplicationApiClient(string baseUri, ILogger<RoatpApplicationApiClie
         {
         }
 
+        //TODO: Will be removed as part of snapshot code cleanup APR-2975
```

Comment:
Delete the TODO comment

Reply:
TODO comment is removed left a pointer comment for APR-2975